vcap/vcap/loading/capsule_loading.py

```diff
@@ -230,40 +232,43 @@ def check_arg_names(func: Callable,
                 pass
 
         # Validate the capsule's backend_loader takes the right args
-        loader = capsule.backend_loader
-        loader_assertions = [
-            callable(loader),
-            check_arg_names(func=loader, correct=["capsule_files", "device"])]
-        if not all(loader_assertions):
+        correct_args = ["capsule_files", "device"]
+        is_correct = check_arg_names(func=capsule.backend_loader,
+                                     correct=correct_args)
+        if not is_correct:
             raise InvalidCapsuleError(
-                f"The capsule's backend_loader has an invalid configuration!\n"
-                f"Loader Assertions: {loader_assertions}",
+                f"The capsule's backend_loader has invalid arguments!\n"
+                f"The arguments must be {correct_args}",
                 capsule.name)
 
         # Validate the backend class attributes
         if capsule.backends is not None:
             backend = capsule.backends[0]
-            backend_assertions = [
-                callable(backend.batch_predict),
-                callable(backend.process_frame),
-                callable(backend.close),
-                isinstance(capsule.backends[0], BaseBackend)]
-            if not all(backend_assertions):
+            _validate_backend_field(capsule,
+                                    "batch_predict",
+                                    backend.batch_predict,
+                                    callable)
+            _validate_backend_field(capsule,
+                                    "process_frame",
+                                    backend.process_frame,
+                                    callable)
+            _validate_backend_field(capsule,
+                                    "close",
+                                    backend.close,
+                                    callable)
+            if not isinstance(capsule.backends[0], BaseBackend):
                 raise InvalidCapsuleError(
-                    f"The capsule's backend has an invalid configuration!\n"
-                    f"Backend Assertions: {backend_assertions}",
+                    f"The capsule's backend field must be a class that "
+                    f"subclasses {BaseBackend.__name__}",
                     capsule.name)
 
         # Validate the stream state
         stream_state = capsule.stream_state
-        stream_state_assertions = [
-            (stream_state is BaseStreamState or
-             BaseStreamState in stream_state.__bases__)]
-        if not all(stream_state_assertions):
+        if not (stream_state is BaseStreamState or
+                BaseStreamState in stream_state.__bases__):
```

Comment:
Would `issubclass` work here?

Reply:
I'm not sure. @apockill do you know why we had to do this?

Reply:
I can't remember :sweat_smile: but I don't think I came to this by accident. An easy way to test would be to change it and run the pharmacy/capsule_zoo tests

Reply:
The tests for both the Pharmacy and Capsule Zoo pass just fine with `issubclass`! I've added it to this PR.